homeassistant/components/bluecurrent/const.py

```diff
@@ -0,0 +1,57 @@
+"""Constants for the BlueCurrent integration."""
+import logging
+
+from homeassistant.const import CONF_DEVICE_CLASS, CONF_NAME, CONF_UNIT_OF_MEASUREMENT
+
+DOMAIN = "bluecurrent"
+
+LOGGER = logging.getLogger(__package__)
+
+PLATFORMS = ["sensor"]
+
+# temp
+URL = "ws://172.18.228.101:8765"
+
+SENSOR_TYPES = {
+    "voltage 1": {
```

Comment:
Voor alle velden hier geldt dat het niet `voltage n` is maar `Voltage phase n` en `current phase n` als je het correct wil verwoorden :)